syntax = "proto3";
import 'nanopb.proto';

message SongDataInternal
{
    repeated uint32 changeLength    = 1 [(nanopb).int_size = IS_8, (nanopb).max_count = 16, (nanopb).fixed_count = true];
    uint32 syncIn                   = 2 [(nanopb).int_size = IS_8];
    uint32 syncOut                  = 3 [(nanopb).int_size = IS_8];
    uint32 bpm                      = 4 [(nanopb).int_size = IS_8];
    uint32 scale                    = 6 [(nanopb).int_size = IS_8];
    uint32 root                     = 7 [(nanopb).int_size = IS_8];
}